src/metabase/models/setting.clj

```diff
@@ -148,9 +148,54 @@
   [_setting]
   [:key])
 
+(def ^:private exported-settings
+  '#{admin-email
+     application-colors
+     application-favicon-url
+     application-font
+     application-font-files
+     application-logo-url
+     application-name
+     available-fonts
+     available-locales
+     available-timezones
+     breakout-bins-num
+     custom-formatting
+     custom-geojson
+     custom-geojson-enabled
+     enable-content-management?
+     enable-embedding
+     enable-nested-queries
+     enable-sandboxes?
+     enable-whitelabeling?
+     enable-xrays
+     hide-embed-branding?
+     humanization-strategy
+     landing-page
+     loading-message
+     map-tile-server-url
+     max-results-bare-rows
+     native-query-autocomplete-match-style
+     persisted-models-enabled
+     report-timezone
+     report-timezone-long
+     report-timezone-short
+     search-typeahead-enabled
+     show-homepage-data
+     show-homepage-pin-message
+     show-homepage-xrays
+     show-lighthouse-illustration
+     show-metabot
+     site-locale
+     site-name
```

Comment:
This is more likely to be customized for deployments vs `application-name` above.